src/web/app/pages-instructor/instructor-search-page/student-result-table/student-result-table.component.ts

```diff
@@ -1,5 +1,22 @@
 import { Component, Input, OnInit } from '@angular/core';
-import { SearchStudentsTable } from '../instructor-search-page.component';
+import { StudentListSectionData } from '../../../components/student-list/student-list-section-data';
+import { StudentListRowModel } from '../../../components/student-list/student-list.component';
+
+/**
+ * Search result for a list of sections containing students of a course
+ */
+export interface SearchStudentsTable {
+  courseId: string;
+  sections: StudentListSectionData[];
```

Comment:
Let's make sure that `StudentListSectionData` is only used in the search service.

Reply:
Yup. `StudentListSectionData` is only used in search service and its test files

Reply:
It is used once in `InstructorSearchPageComponent` because the search service currently returns `StudentListSectionData` and it is flattened in the component. Is that okay?